api-extractor.json

```diff
@@ -19,7 +19,7 @@
 
   "dtsRollup": {
     "enabled": true,
-    "untrimmedFilePath": "docs/generated/api.d.ts",

```

Comment:
After changing this, is the path for this file the same?

https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/docs/generated/api.d.ts

Reply:
no, it actually overwrites `build/src/index.d.ts`

Reply:
OK. Are you going to keep both copies in `docs` and `build`, or how are we expecting `docs/generated/api.d.ts` to keep being updated after this change?

Reply:
ah, I see your point. I will delete the `docs/generated/api.d.ts`